packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmTests.kt

```diff
@@ -635,6 +637,118 @@ class RealmTests {
         }
     }
 
+    @Test
+    fun assetFile() {
+        val config = RealmConfiguration.Builder(setOf(Parent::class, Child::class))
+            .directory(tmpDir)
+            .log(LogLevel.DEBUG)
+            // Need a separate name to avoid clashes with this.realm
+            .name("prefilled.realm")
+            .schemaVersion(1)
```

Comment:
Is this needed?

Reply:
Apparently not. It was somehow related to the way that I constructed the `asset.realm`. I have recreated it and now the schema version is in place without the need for this.